src/restapi/fHelp.ts

```diff
@@ -29,35 +31,38 @@ function getCommands(call: AppCallRequest): string {
 }
 
 function getUserCommands(): string {
-    let text = h5('User Commands');
-    text += addBulletSlashCommand('connect');
-    text += addBulletSlashCommand('disconnect');
-    text += addBulletSlashCommand('help');
-    return text;
+    return [
+        h5('User Commands'),
+        addBulletSlashCommand('connect'),
+        addBulletSlashCommand('disconnect'),
+        addBulletSlashCommand('help'),
+    ].join('\n') + '\n';
 }
 
 function getAdminCommands(): string {
-    let text = h5('System Admin Commands');
-    text += addBulletSlashCommand('configure');
-    text += addBulletSlashCommand('setup-target');
-    text += addBulletSlashCommand('subscribe');
-    return text;
+    return [
+        h5('System Admin Commands'),
+        addBulletSlashCommand('configure'),
+        addBulletSlashCommand('setup-target'),
+        addBulletSlashCommand('subscribe'),
+    ].join('\n') + '\n';
 }
 
 function getPostText(): string {
-    let text = h5('Post Menu Options');
-    text += 'click the (...) on a post\n';
-    text += addBullet('Create Zendesk Ticket');
-    text += addBullet('Zendesk Subscriptions');
-    return text;
+    return [
+        h5('Post Menu Options'),
+        'click the (...) on a post\n',
```

Comment:
Maybe make a `text` function that appends the newline so it's more consistent with the other things in this list. Also, should "click" be capitalized?

Reply:
done.